doc/api/buffer.md

```diff
@@ -415,6 +415,21 @@ function:
 * [`Buffer.from(arrayBuffer[, byteOffset[, length]])`][`Buffer.from(arrayBuf)`]
 * [`Buffer.from(string[, encoding])`][`Buffer.from(string)`]
 
+### Buffer methods are callable with `Uint8Array` instances
```

Comment:
I'd add a `changes:` YAML entry here that lists all the methods for which behavior actually changed, as well as for the individual methods where this behavior has been modified, so that it's easier for users to track which Node.js versions support this behavior and which don't